byoqm/visuals/dashboard.py

```diff
@@ -25,6 +29,8 @@ def get_data(self, path="./output"):
         graph_data = defaultdict(list)
         for filename in os.listdir(path):
             date = datetime.strptime(filename.split(".")[0], "%Y-%m-%d_%H-%M-%S")
+            if self._start_date > date or self._end_date < date:
+                continue
```

Comment:
I would argue that we should add tests targeting this if statement because it is my experience is that dates are always a pain to work with.